config/kibana.yml

```diff
@@ -61,6 +61,11 @@
 # must be a positive integer.
 # elasticsearch.requestTimeout: 30000
 
+# List of Kibana client-side headers to send to Elasticsearch. To send *no* client-side
+# headers, set this value to null, not []. Setting it to [] will send the default list
```

Comment:
If you pull master you should see that `elasticsearch.requestHeadersWhitelist: []` now clears the whitelist.